backend/src/v5/processors/teamspaces/projects/models/federations.js

```diff
@@ -32,10 +32,11 @@ const { queueFederationUpdate } = require('../../../../services/modelProcessing'
 const Federations = { ...Groups, ...Views, ...Tickets, ...Comments, ...TicketGroups };
 
 // Override
-Federations.getTicketGroupById = async (teamspace, project, federation, revId, ticket, groupId) => {
+Federations.getTicketGroupById = async (teamspace, project, federation, revId, ticket, groupId,
+	convertTo3DRepoGuids) => {
```

Comment:
rename `convertTo3DRepoGuids` to something similar in the ticket groups processor suggestion